src/main/java/com/exasol/release/ReleaseMakerFactory.java

```diff
@@ -18,17 +19,20 @@ private ReleaseMakerFactory() {
     /**
      * Create a new instance of the {@link ReleaseMaker} depending on the {@link Platform}.
      * 
-     * @param content {@link GitBranchContent} with the project content to release
-     * @param platform release platform
+     * @param content       {@link GitBranchContent} with the project content to release
+     * @param platform      release platform
+     * @param releaseReport release report
      * @return new instance of {@link ReleaseMaker}
      */
-    public static ReleaseMaker createReleaseMaker(final GitBranchContent content, final Platform platform) {
+    public static ReleaseMaker createReleaseMaker(final GitBranchContent content, final Platform platform,
+            final ReleaseReport releaseReport) {
         final PlatformName platformName = platform.getPlatformName();
         if (platformName == GITHUB) {
-            return new GitHubReleaseMaker(content, (GitHubPlatform) platform);
+            return new GitHubReleaseMaker(content, (GitHubPlatform) platform, releaseReport);
         } else {
-            throw new UnsupportedOperationException("E-RR-REL-1: Release for platform " + platform
-                    + " is not supported. Please choose one of: github");
+            // TODO: remove hardcoded list of supported platforms
+            throw new UnsupportedOperationException(
+                    "E-RR-REL-1: Release for platform " + platform + " is not supported. Please choose one of: github");
```

Comment:
quote platform

Reply:
It's dangerous, What if people try to insert them together with quotes into parameters? The parsing will fail then